src/test/java/fi/helsinki/cs/tmc/core/updates/ExerciseUpdaterTest.java

```diff
@@ -20,19 +20,12 @@
 import fi.helsinki.cs.tmc.core.testhelpers.ExampleJson;
 import fi.helsinki.cs.tmc.core.testhelpers.builders.ExerciseBuilder;
 
-import com.google.gson.Gson;
-
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import org.mockito.Mockito;
-
-import java.io.File;
-import java.io.FileWriter;
 import java.io.IOException;
+import java.net.URI;
 import java.net.URISyntaxException;
-import java.nio.file.Paths;
```

Comment:
The import issues are from the automatic import of NetBeans. Is there a way to configure it so that it would automatically add the imports to their correct locations? It seems tedious having to do this by hand for each file.